src/publisher.ts

```diff
@@ -264,11 +264,21 @@ export const publishEmployeesCelebrations = async (
         ],
       }))
     );
-
+  } else {
     birthdaysBlocks.push({
-      type: 'divider',
+      type: 'context',
+      elements: [
+        {
+          type: 'mrkdwn',
+          plain_text:
+            ':mad-hatter::teapot: *A very Merry Unbirthday to you all!* :mad-hatter::teapot:',
+        },
+      ],
     });
   }
+  birthdaysBlocks.push({
+    type: 'divider',
+  });
```

Comment:
@totegsito , I love the message to be displayed :) 

My only concern is that I doubt whether we should only display the message if there are no celebrations at all (birthdays, anniversaries or first days). What do you think?

Reply:
That was my first approach, but then I realized that having a default message in case there is no birthday means that always there is something to celebrate, which is funnier than a default message.

I had some doubts between this and something "inspirational" when there is nothing to celebrate, but... everybody likes Alice in wonderland 😬

Reply:
If you prefer to apply this message just in case there is nothing to celebrate, let me know and I will change it quickly ⚡

Reply:
I really like that 'Inspirational' approach.
We could use a random Inspirational quote when there's nothing special to celebrate!

Reply:
i.e. https://zenquotes.io/  why not?

Reply:
changed on [4a93450](https://github.com/guidesmiths/bamboohrbot/pull/14/commits/4a934507c8634a8e84790546d0133f96bcfe8261)

Reply:
@Betisman I changed it so it shows the same message always when there is nothing to celebrate. 
I did not know that API exists and actually I was going to use some hardcoded phrases. 

What if we create a new issue for the inspirational approach? Honestly I see that as something cool to show everyday, not just in case there is no values to show

Reply:
Yeah, it may be a good idea :) 

Let's merge it, and see how it works!

Reply:
@totegsito , issue added :)

https://github.com/guidesmiths/bamboohrbot/issues/15